packages/generators/init-generator.ts

```diff
@@ -40,16 +43,63 @@ export default class InitGenerator extends Generator {
 		super(args, opts);
 		this.isProd = false;
 		this.usingDefaults = false,
+
 		this.dependencies = [
 			"webpack",
 			"webpack-cli",
-			"terser-webpack-plugin",
```

Comment:
why is terser removed

Reply:
It gets added later only in production. Makes sense

Reply:
I agree with @ematipico.

I'm confused because in the [current](https://github.com/webpack/webpack-cli/blob/master/packages/generators/init-generator.ts#L430) code we remove terser in production and we include it only in devmode (also, we never use the plugin in the configuration) 

```typescript
if (this.isProd) {
	this.dependencies = this.dependencies.filter(
		(p: string): boolean => p !== "terser-webpack-plugin",
	);
} else {
	this.configuration.config.topScope.push(
		tooltip.terser(),
		"const TerserPlugin = require('terser-webpack-plugin');",
		"\n",
	);
}
```

Is terser only in production the way to go?

Reply:
Yes! It doesn't make much sense having it in production